scripts/buildkite/golint.sh

```diff
@@ -15,5 +15,6 @@ if [ -n "$(git status --porcelain  2>&1)" ]; then
   echo "There are changes after make go-generate && make fmt && make lint && make copyright"
   echo "Please rerun the command and commit the changes"
   git status --porcelain
+  git --no-pager diff
```

Comment:
this seem unrelated?

Reply:
`git status --porcelain` was showing me there were uncommitted changes but it doesn't show what those changes are to quickly address missing things. `git diff` will print those changes